src/config.yaml

```
@@ -29,7 +29,7 @@ canopen_vcan:
         # Implementation saves raw can messages up until buffer's value. 
         buffer: 1024,
         # At every n-th message update shared memory
-        update_shm: 100
+        update_shm_threshold: 100
```

moeten we dit niet ergens bovenin de code als define hebben, zodat we niet de code in hoef te duiken telkens?
Dus:
```
UPDATE_THRESHOLD = 100
...
```
```suggestion
        update_shm_threshold: UPDATE_THRESHOLD
```

---

Nevermind, ik had niet door dat dit de config was